.env.development

```diff
@@ -1,12 +1,12 @@
 REACT_APP_URI=http://localhost:3000/
 REACT_APP_ANUBIS_API_URL=http://localhost:8085/
 REACT_APP_CONFIGURATION_API_URL=http://localhost:4000/configuration
-REACT_APP_OIDC_ISSUER=http://keycloak:8080/realms/default
+REACT_APP_OIDC_ISSUER=http://localhost:8080/realms/default
```

Comment:
if you use localhost tenant creation will fail, no?

Reply:
no, with your configuration i was not able to login on keyckoack